acceptance/apps_test.go

```diff
@@ -404,6 +404,14 @@ var _ = Describe("Apps", func() {
 				Expect(err).ToNot(HaveOccurred(), out)
 				return out
 			}, "1m").ShouldNot(MatchRegexp(`.*%s.*`, appName))
+
+			By("checking if servicebinding is deleted")
+			serviceBindings, err := helpers.Kubectl("get", "servicebindings",
+				"--namespace", org)
+			Expect(err).NotTo(HaveOccurred())
+			Expect(serviceBindings).NotTo(ContainSubstring(fmt.Sprintf("svc.org-%s.svc-%s.app-%s", org, serviceName, appName)))
```

Comment:
The tests have a `helper/catalog/names.go` file (`catalog` package). Might be sensible to put the `Sprintf` into a function there to hide the details of the construction here.

Reply:
Done.